cnf-app-mac-operator/config/manager/manager.yaml

```diff
@@ -35,5 +35,30 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 6789
+          initialDelaySeconds: 15
+          periodSeconds: 20
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: 6789
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        startupProbe:
+          httpGet:
+            path: /startupz
+            port: 6789
+          initialDelaySeconds: 5
+          periodSeconds: 10
         terminationMessagePolicy: FallbackToLogsOnError
+        lifecycle:
+          postStart:
+            exec:
+              command: ["/bin/sh", "-c", "echo Hello from the postStart handler"]
+          preStop:
+            exec:
+              command: ["/bin/sh", "-c", "killall -0 tail"]
```

Comment:
Is the container running a tail process? If not this will fail

Reply:
I've copied this from another example, I'll change this for an echo, this is just to have a valid preStop condition. Thanks!